<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>421</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_3_tai_khoan_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ECE3D2;
color: #356A9C;
font: 12pt &quot;Segoe UI&quot;;
border-radius: 15px;
padding-left: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Tài Khoản</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Bahnschrift&quot;;
color: #575556;</string>
     </property>
     <property name="text">
      <string>Họ và tên</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton2_dangki">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>480</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Bahnschrift&quot;;
color: #356a9c;
background-color: rgba(255, 0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Đăng Nhập Ngay</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_DangKi">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>410</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
font: 600 16pt &quot;Bahnschrift&quot;;
background-color: #356a9c;
color: #F8F2EC;</string>
     </property>
     <property name="text">
      <string>Đăng Kí</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3_tai_khoan">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ECE3D2;
color: #356A9C;
font: 12pt &quot;Segoe UI&quot;;
border-radius: 15px;
padding-left: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Họ và tên</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2_mat_khau">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ECE3D2;
color: #356A9C;
font: 12pt &quot;Segoe UI&quot;;
border-radius: 15px;
padding-left: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>32772</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>Mật Khẩu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Bahnschrift&quot;;
color: #575556;</string>
     </property>
     <property name="text">
      <string>Mật khẩu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #4d0ffa;
font: 20pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>Đăng Kí</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Bahnschrift&quot;;
color: #575556;</string>
     </property>
     <property name="text">
      <string>Tài Khoản</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>471</width>
      <height>591</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>591</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../img/3766277.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
